clients/privacy-center/components/modals/privacy-request-modal/PrivacyRequestForm.tsx

```diff
@@ -64,7 +65,15 @@ const usePrivacyRequestForm = ({
   const formik = useFormik<FormValues>({
     initialValues: {
       ...Object.fromEntries(
-        Object.entries(identityInputs).map(([key]) => [key, ""])
+        Object.entries(identityInputs)
+          .filter(
+            ([key, value]) =>
+              key === "name" ||
+              key === "phone" ||
+              key === "email" ||
+              (typeof value === "object" && value.label)
+          )
```

Comment:
Only keeping the known identities (name, phone, email) or custom identities with a label, this is to ignore something just doing
```
"loyality_id: "required"
```
it needs to be labeled
```
"loyalty_id": {"label" "Loyalty ID"}
```

Reply:
The repeated list of keys here would benefit from some kind of constant declaration like:
```
const DEFAULT_IDENTITY_INPUTS_MAP = {
  "name": undefined,
  "email": "email",
  "phone": "phone_number",
};
```
...which could more easily do comparisons like `Object.keys(DEFAULT_IDENTITY_INPUTS_MAP).includes("name")` etc